### ISSUE TITLE
change toBe in reporting to `equals`

### ISSUE THREAD

**Issue by robstoll** [robstoll/atrium#308]
*Platform* (jvm, js, android): all
*Extension* (none, kotlin 1.3, jdk8): none

## Code related feature
```
expect(1).toBe(1)
```
results in 
```
expect: 1
* to be: 2
```

we would like to change it to:
```
expect: 1
* equals: 2
```

same same but different for notToBe with `does not equal` in reporting

*translations*
- [ ] change DescriptionBasic accordingly for en_GB

## Your first contribution?
- Write a comment `I'll work on this` if you would like to take this issue over. 
  This way we get the chance to revise the description in case things have changed in the meantime, we might give you additional hints and we can assign the task to you, so that others do not start as well.
- See [Your first code contribution](https://github.com/robstoll/atrium/blob/master/.github/CONTRIBUTING.md#your-first-code-contribution) for guidelines.  
- Do not hesitate to ask questions here or to contact us via [Atrium's slack channel](https://kotlinlang.slack.com/team/U3DE1TXKP) if you need help
  ([Invite yourself](https://slack.kotlinlang.org/) in case you do not have an account yet).

**Comment by Hubisco** [comment]
I'll work on this :)

**Comment by robstoll** [comment]
I realised that `is equal to` is not ideal as there are types which implement an `isEqual` function (LocalDate etc.) and in such cases it would be misleading not clear which behaviour no was asserted against - Object.equals or LocalDate.isEqual. Therefore I decided that we use `equals` instead which resembles more the code.

**Comment by robstoll** [comment]
implemented with 0bf1327
